### PR TITLE
Dev 56 - different colours for the story labels

### PR DESCRIPTION
**Pull request by KovDimaY**
Please make sure these boxes are checked before submitting your PR, thank you!

* [x] Make sure all the tests are still passing.
* [x] Make sure the test coverage is still 100%.
* [x] Make sure your source branch is correct.
* [x] Make sure your destination branch is **devel**.
* [x] Make sure the name of the PR has a correct DEV-# identification prefix.
* [x] Make sure you reviewed all your changes and they look correct.
* [x] Make sure all the commits have a correct format. 
* [x] Make sure you refer related issues for you PR. 
* [x] Make sure you add **KovDimaY** as a reviewer and assignee of your PR.

Take a look at the Issue #103

### CODE REVIEW

File: client/src/components/Common/DetailsLabel/constants.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  ok: '#FEC22D',
+  bad: '#F47048',
+  very_bad: '#E91E63',
+};
+
+const limitsExpences = [100, 500, 1000, 5000];
+const limitsDuration = [2, 7, 14, 31];
+const limitsRating = [1, 2, 3, 4];
+
+const getColorsInversed = (value, limits) => {
```
Comment (NikolayYakovenko) [inline client/src/components/Common/DetailsLabel/constants.js]: It is not constants :)
Move these functions to helpers.

File: client/src/components/Common/DetailsLabel/constants.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
+export const TYPES = {
+  expences: 'EXPENCES',
+  duration: 'DURATION',
+  rating: 'RATING',
+};
+
+const colors = {
+  very_good: '#00B300',
```
[the remark this answers is not in the document]
Comment (KovDimaY) [reply client/src/components/Common/DetailsLabel/constants.js]: Well, actually it is a question of preference, but yes, I think you are right, with camelCase it looks better... 🤔 
Thanks! 😇

File: client/src/components/Common/DetailsLabel/constants.js (hunk 1 above)
Comment (KovDimaY) [reply client/src/components/Common/DetailsLabel/constants.js]: Well, it is true... But as I would like to keep all this logic in the same file, I am going to rename all the file to "constants.js" ---> "utils.js". 

What do you think?

Comment (NikolayYakovenko) [reply client/src/components/Common/DetailsLabel/constants.js]: Yes, good idea! I think, your new solution is more appropriate.

**Comment by NikolayYakovenko** [comment]
Good job!

**Pull request merged**